### DISCUSSION TITLE
Environment variables from `auto_envvar` don't show up in docs

### DISCUSSION THREAD

**Discussion by danielgafni** [tiangolo/typer-cli#143] [Questions] [answered]
It would be useful to have them there!

**Comment by tiangolo** [comment, marked as answer]
Thanks! Typer-CLI is now integrated into Typer, so everything is in that repo. Now you just need to install with `pip install typer` and you get the `typer` command and everything.

Could you please add this feature request to the Typer repo in a new discussion question?: https://github.com/tiangolo/typer/discussions/new?category=questions
